pkg/piperutils/container.go

```diff
@@ -0,0 +1,41 @@
+package piperutils
+
+import (
+	"fmt"
+	"net/url"
+	"strings"
+
+	"github.com/google/go-containerregistry/pkg/name"
```

Comment:
What about a more speaking alias?

```suggestion
	containerName "github.com/google/go-containerregistry/pkg/name"
```